src/GitVersion.Core.Tests/Formatting/DateFormatterTests.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System.Globalization;
 using GitVersion.Formatting;
 
-namespace GitVersion.Tests.Formatting;
+namespace GitVersion.Core.Tests.Formatting;
```

Comment:
`GitVersion.Core` should not be a namespace. :)

```suggestion
namespace GitVersion.Tests.Formatting;
```